### PR TITLE
event update

### PR DESCRIPTION
**Pull request by talhamalik883**

### CODE REVIEW

File: contracts/smart-contract-wallet/SmartWallet.sol [hunk 1]
```diff
@@ -68,8 +68,9 @@ contract SmartWallet is
 
     function setOwner(address _newOwner) external mixedAuth {
         require(_newOwner != address(0), "Smart Account:: new Signatory address cannot be zero");
+         address oldOwner = owner;
         owner = _newOwner;
-        emit EOAChanged(address(this),owner,_newOwner);
+        emit EOAChanged(address(this), oldOwner, _newOwner);
```
Comment (livingrockrises) [inline contracts/smart-contract-wallet/SmartWallet.sol:73]: if newOnwer is this way : _newOwner
review name of oldOwner. Should it not be _oldOwner ??

**Review by livingrockrises — COMMENTED** [review]
check comments

File: contracts/smart-contract-wallet/SmartWallet.sol (hunk 1 above)
Comment (talhamalik883) [reply contracts/smart-contract-wallet/SmartWallet.sol:73]: variable name starting with _ is the convention only needs to be followed for arguments. Variable inside the class or function scope does not needs to be declared with _

**Review by livingrockrises — APPROVED** [review]
approved

**Pull request merged**